workflows/standalone_modules/wf_kraken2_ont.wdl

```diff
@@ -0,0 +1,42 @@
+version 1.0
+
+import "../../tasks/taxon_id/task_kraken2.wdl" as kraken2
+import "../../tasks/task_versioning.wdl" as versioning
+
+workflow kraken2_ont_wf {
+  meta {
+    description: "Classify ONT single-end reads using Kraken2 by recalculating abundances based on read length"
+  }
+  input {
+    String samplename
+    File read1
+    File kraken2_db
+  }
+  call kraken2.kraken2_standalone as kraken2_se {
+    input:
+      samplename = samplename,
+      read1 = read1,
+      kraken2_db = kraken2_db
+  }
+  call kraken2.kraken2_parse_classified as kraken2_recalculate_abundances {
+    input:
+      samplename = samplename,
+      kraken2_report = kraken2_se.kraken2_report,
+      kraken2_classified_report = kraken2_se.kraken2_classified_report
+  }
+  call versioning.version_capture{
+    input:
+  }
+  output {
+    # PHB Version Captures
+    String kraken2_se_wf_version = version_capture.phb_version
+    String kraken2_se_wf_analysis_date = version_capture.date
+    # Kraken2
+    String kraken2_version = kraken2_se.kraken2_version
+    String kraken2_docker = kraken2_se.kraken2_docker
+    File kraken2_report = kraken2_recalculate_abundances.kraken_report
+    File kraken2_classified_report = kraken2_se.kraken2_classified_report
```

Comment:
yeah i'd vote to exclude the classified report from the outputs

Reply:
it is! renamed the variable to be more accurate

Reply:
Done!

Reply:
Going back on this, this output is part of the kraken_SE and kraken_PE standalone outputs. Should it also be removed from those? Or should I keep this output in all the workflows (SE, PE and ONT)?

Reply:
I vote to remove from SE & PE standalone.